_posts/2022-04-20-WildFly-s2i-v2-Released.adoc

```diff
@@ -0,0 +1,232 @@
+---
+layout: post
+title:  "WildFly S2I new architecture is final!"
+date:   2022-04-20
+tags:   wildfly galleon cloud wildfly-s2i openshift
+author: jfdenise
+---
+
+== New WildFly on the cloud architecture
+
+We have released the new components that make for a completely redesigned WildFly cloud experience. 
+This redesign allows for more flexibility, simpler workflows and smaller application images.
+
+Our new cloud architecture is composed of 5 main components:
+
+* New WildFly S2I builder images (link:https://quay.io/repository/wildfly/wildfly-s2i-jdk11[JDK11] and link:https://quay.io/repository/wildfly/wildfly-s2i-jdk17[JDK17])
+* New WildFly runtime images (link:https://quay.io/repository/wildfly/wildfly-runtime-jdk11[JDK11] and link:https://quay.io/repository/wildfly/wildfly-runtime-jdk17[JDK17])
+* New link:https://github.com/wildfly/wildfly-maven-plugin/[WildFly Maven plugin 3.0.0.Final] 
+* New link:https://github.com/wildfly-extras/wildfly-cloud-galleon-pack[WildFly cloud Galleon feature-pack 1.0.0.Final] 
+* New link:https://github.com/wildfly/wildfly-charts[Helm Chart for WildFly 2.0] 
+
+The new S2I workfow can be summerized as:
+
+* Configure the WildFly Maven plugin to deploy your application in the WildFly server of your choice, finely tuned according to your needs (link:https://github.com/wildfly/wildfly-s2i/blob/main/examples/jsf-ejb-jpa/pom.xml[an example])!
+* Make your application sources available in a git repository (link:https://github.com/wildfly/wildfly-s2i/blob/main/examples/postgresql[an example]).
+* Define a Helm Chart for WildFly yaml file (link:https://github.com/wildfly/wildfly-s2i/blob/main/examples/postgresql/helm.yaml[an example]).
+* Install your helm file into OpenShift (link:https://github.com/wildfly/wildfly-s2i/blob/main/examples/jsf-ejb-jpa/README.md#example-steps[an example]). You are done!
+
+To get you familiar with the various components you can directly jump to these link:https://github.com/wildfly/wildfly-s2i/tree/main/examples[examples]. 
+They will drive you through various use-cases from source to deployment on OpenShift (for immediate access to an OpenShift cluster, you can use link:https://developers.redhat.com/developer-sandbox[OpenShift sandbox]).
+
+== Legacy WildFly S2I and runtime centos7 images
+
+We have released the legacy images for WildFly 26.1.0 in link:https://quay.io/organization/wildfly[quay.io/wildfly]. 
```

Comment:
either "WildFly 26.1" or "WildFly 26.1.0.Final" would be more appropriate

Reply:
Fixed.